electron/index.d.ts

```diff
@@ -1864,12 +1864,7 @@ declare namespace Electron {
 		/**
 		 * Writes data to the clipboard.
 		 */
-		write(data: {
-			text?: string;
-			rtf?: string;
-			html?: string;
-			image?: NativeImage;
-		}, type?: ClipboardType): void;
+		write(data: { text: string; bookmark?: string; } | { rtf: string; } | { html: string; } | { image: NativeImage; }, type?: ClipboardType): void;
```

Comment:
The type with all optional properties is equivalent to a mere `any`, and allows to write `clipboard.write("test_string")`. See the details I wrote in the first post of this PR.